packages/yarnpkg-cli/sources/boot-cli-dev.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+require('../../../.pnp.js').setup();
+
+process.env.NODE_OPTIONS += ` --require ${'../../../.pnp.js'} --require ${require.resolve(`@yarnpkg/monorepo/scripts/setup-ts-execution`)}`;
+
+require(`@yarnpkg/monorepo/scripts/setup-ts-execution`);
+global.YARN_VERSION = require('@yarnpkg/core').version;
+const dynamicLibs = require('@yarnpkg/builder/sources/data/dynamicLibs').dynamicLibs;
+const plugins = require('@yarnpkg/builder/sources/tools/findPlugins.ts').findPlugins({basedir: path.dirname(require.resolve(`@yarnpkg/cli/package.json`)), plugins: []});
+
+const PLUGIN_CONFIG_MODULE = './pluginConfiguration.raw.js';
+require(PLUGIN_CONFIG_MODULE);
```

Comment:
Do you actually need to `require` it? Since the cache entry is reset right after we can just set it directly to `{exports: ...}`